test/start-operator.sh

```diff
@@ -75,6 +75,10 @@ function deploy_using_olm() {
 }
 
 function deploy_using_yaml() {
+  crd=${REPO_DIRECTORY}/deploy/crd/pmem-csi.intel.com_deployments.yaml
+  echo "Deploying '${crd}'..."
+  cat  ${crd} | ${SSH} kubectl create -f -
```

Comment:
Better use `apply`, as for the operator itself. Sometimes that makes it simpler to re-deploy.

Reply:
Change as suggested.